app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```

Comment:
@apeksha2697  please remove commented code from this file.
Ideally your PR should never contain commented code

Reply:
done